tests/lib/snaps/snapctl-hooks/meta/hooks/configure

```diff
@@ -67,7 +67,9 @@ test_get_nested() {
       exit 1
   fi
   expected_output='{\n\t"key1": "a",\n\t"key2": "b"\n}'
-  if ! ( echo "$output" | grep -z -q -P "$expected_output" ); then
+  # note: "echo" is a built-in of sh and doesn't support -e flag, use /bin/echo.
```

Comment:
We can use `command echo` to execute the actual external program, not the shell built-in.

Reply:
hmm, actually.. that doesn't work in sh, a quick test:

```
$ sh
$ command echo -e "foo"
-e foo
```
I'm inclined to leave the above as is after spending too much time on it in previous PR already ;)